pyvo/dal/sia2.py

```diff
@@ -156,7 +156,7 @@ class SIA2Service(DALService, AvailabilityMixin, CapabilityMixin):
     generally not notice that, though.
     """
 
-    def __init__(self, baseurl, session=None):
+    def __init__(self, baseurl, session=None, check_baseurl=True):
```

Comment:
Seaparate, but I would like to make optional arguments keyword only, systematically, and before the 1.5 release.